CONTRIBUTING.md

```diff
@@ -51,11 +51,13 @@ If your problem relates to sensitive or private information, please don't post a
 Follow these steps to test your changes
```

Comment:
We should break title this section contributing to the core components library, or similar, since this is a pretty different path now than if you wanted to change a db connection.